public/section1.js

```diff
@@ -0,0 +1,73 @@
+(function() {
+    // init navigation dot
+    let nav1 = document.getElementById("nav1");
+    let nav1Dot = nav1.children[0];
+    nav1Dot.classList.add("active");
+
+    // click nav -> turn nav white and fo to id
+    let nav2 = document.getElementById("nav2");
+    let nav2Dot = nav2.children[0];
+    let nav3 = document.getElementById("nav3");
+    let nav3Dot = nav3.children[0];
+
+    nav1.addEventListener('click', () => {
+        console.log('done1');
+        if (nav1Dot.className === 'inner-circle') {
+            nav1Dot.classList.add("active");
+            nav2Dot.classList.remove("active");
+            nav3Dot.classList.remove("active");
+            window.location.hash='one';
+        }
+    });
+    nav2.addEventListener('click', () => {
+        console.log('done2');
+        if (nav2Dot.className === 'inner-circle') {
+            nav2Dot.classList.add("active");
+            nav1Dot.classList.remove("active");
+            nav3Dot.classList.remove("active");
+            window.location.hash='two';
+        }
+    });
+    nav3.addEventListener('click', () => {
+        console.log('done3');
+        if (nav3Dot.className === 'inner-circle') {
+            nav3Dot.classList.add("active");
+            nav1Dot.classList.remove("active");
+            nav2Dot.classList.remove("active");
+            window.location.hash='three';
+        }
+    });
+
+    var scrollMenu = function() {
```

Comment:
Use const instead of var

Reply:
Thank you
fixed on this commit https://github.com/eastend-street/quote-maker/pull/7/commits/62f4a3752f4702b18915a2b3121f344c7b028f77

Reply:
Okay, thanks! Wait a moment, still checking other code..